lib/ex_doc/formatter/epub/templates/nav_template.eex

```diff
@@ -1,5 +1,5 @@
-<%= head_template(config, %{title: "Table Of Contents"}) %>
-    <h1>Table of contents</h1>
+<%= head_template(config, %{title: translate(config.language, "Table Of Contents")}) %>
+    <h1><%= translate(config.language, "Table of contents") %></h1>
```

Comment:
We should unify both of these translations into one. We can use "Table of contents" for both.